src/docs/5.33.x/headless-cms/extending/content-models-via-code.mdx

```diff
@@ -375,3 +375,25 @@ Here are a couple of examples of specifying icons from solid, regular, and brand
 - `far/copy`
 - `fab/aws`.
 - `fab/react`.
+
+## Immutable Code Definitions
+
+As you can do what ever you want with the models via code, there are things you need to be really careful about.
+We cannot prevent you to change anything, so you need to be watchful when creating models via code.
+
+You should **never** change `modelId` property of the model definition.
+
+There are also few field properties which must **never** change, if you want to keep your data.
+
+Properties are:
+ - `id`
+ - `storageId` - if you have created it manually
+ - `type`
+ - `multipleValues`
+ - `predefinedValues` - you can add values but do not remove them
+
+Also, you have defined anything settings, be careful about these properties:
+
+ - `settings.fields` - same rule applies as for the parent field
```

Comment:
Same what rule? What's the parent?

Reply:
parent field being the one you have defined settings.fields in?